src/lib/core/authorization/AuthorizationManager.ts

```diff
@@ -351,7 +368,10 @@ export class AuthorizationManager {
    * Process a well-formed Authorization Requirements error response from a Globus service
    * and redirect the user to the Globus Auth login page with the necessary parameters.
    */
-  handleAuthorizationRequirementsError(response: AuthorizationRequirementsError) {
+  handleAuthorizationRequirementsError(
+    response: AuthorizationRequirementsError,
+    additionalParams: IObject = {},
```

Comment:
`handleAuthorizationRequirementsError` and `handleConsentRequiredError` should take an  `options` argument with `additionalParameters` as a property so we don't run into issues adding new options in the future. 

```suggestion
    options?: { additionalParams?: IObject }
```